numpy/lib/tests/test_shape_base.py

```diff
@@ -3,14 +3,72 @@
 import numpy as np
 from numpy.lib.shape_base import (
     apply_along_axis, apply_over_axes, array_split, split, hsplit, dsplit,
-    vsplit, dstack, column_stack, kron, tile
+    vsplit, dstack, column_stack, kron, tile, take_along_axis
     )
 from numpy.testing import (
     run_module_suite, TestCase, assert_, assert_equal, assert_array_equal,
     assert_raises, assert_warns
     )
 
 
+class TestTakeAlongAxis(TestCase):
+    def test_argequivalent(self):
```

Comment:
I'd argue for postponing that to #8710 as well, and then no special handling would be needed